packages/tao/src/commands/migrate.ts

```diff
@@ -530,7 +514,7 @@ function updatePackageJson(
       json.dependencies[p] = updatedPackages[p].version;
     }
   });
-  writeFileSync(packageJsonPath, JSON.stringify(json, null, 2) + endOfFile);
+  writeJsonFile(packageJsonPath, json);
 }
```

Comment:
@meeroslav The end of line behaviour was introduced in #3642 as fix for #3628. I think the eol characters and formatting should be handled by prettier. Maybe the migrate command should run prettier or suggest running it as fix for #3628 but it should not be handled by the updatePackageJson method.

Reply:
Hmm... I will look into it. Either way, package.json should keep the same format after migration.
Thanks!

Reply:
I will fix it for now in a separate branch, and we can get back to it later. IMHO running prettier on every file write is not optimal. 

The end-user solution might not use `prettier`, but a simple `.editorconfig` or even just shared IDE configurations.